kiwi/schema/kiwi.rnc

```diff
@@ -1388,6 +1388,15 @@ div {
             sch:param [ name = "attr" value = "filesystem" ]
             sch:param [ name = "types" value = "vmx oem pxe" ]
         ]
+	k.type.squashfscompression.attribute = 
+        ## Specifies the compression type for mksquashfs
+        attribute squashfscompression {
+            "none" | "gzip" | "lzo" | "lz4" | "xz"
+        }
```

Comment:
Do we want "none" to be handled as a value ? I'm asking because this kind of conflicts with none handling in python from a  readers perspective. Technically it's all ok but reading 'none' usually does not imply to be an option setting.
I thought about 'uncompressed" or "raw" or "flat" ... something in that area. Thoughts ?

Reply:
Good point, I did not think about it. I think the `uncompressed` is the best one, long but clear and explicit, gonna change to that.